### PR TITLE
Try fixing the tabs by using the id prop

### LINKED ISSUE #1425: Wrong format displayed in the Credit the creator box
## Description
When opening an URL to see the single result view of any content, tabs in Credit the creator box show an incorrect format of the attribution information. Rich text tab shows HTML content, HTML tab shows plain text, and Plain text shows nothing. 

This does not happen when opening a result from the search results page.

## Reproduction
1. Open [this result](https://search-staging.openverse.engineering/image/0fc8d459-6c59-4370-ae20-cc83c4a64e65).
2. Play with the tabs of the **Credit the creator** box.
3. See the error

## Screenshots
CleanShot 2022-05-12 at 12 08 29

## Environment
 - [Staging](https://search-staging.openverse.engineering)
 - Device:Macbook Pro M1 13-inch
 - OS: MacOS Monterey 12.3.1
 - Browser: Firefox
 - Version: 100.0

## Resolution
- [ ] 🙋 I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Fixes #1425 by @panchovm

## Description

This PR uses the required `tab` prop to link VTab to VPanel and to make sure that the selected tabpanel is set correctly. Now, it is required to use the same `id` for the VTab and VPanel that are linked.

## Testing Instructions

Merge and see...

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/VMediaInfo/VCopyLicense.vue [hunk 1]
```diff
@@ -13,20 +13,22 @@
...
-      <VLicenseTabPanel :tab="tabs[1]">
-        <p id="attribution-html" class="font-mono break-all" dir="ltr">
-          {{ getAttributionMarkup() }}
-        </p>
-      </VLicenseTabPanel>
-      <VLicenseTabPanel :tab="tabs[2]">
-        {{ getAttributionMarkup({ isPlaintext: true }) }}
-      </VLicenseTabPanel>
+      <template #default="{ selectedTabId }">
+        <VLicenseTabPanel :tab="tabs[0]" :selected="selectedTabId === 0">
```
Comment (sarayourfriend) [inline src/components/VMediaInfo/VCopyLicense.vue:17]: Given tabs have an ID already (the value passed to `tab` prop) why do we use the index? It seems potentially flaky if a Tab component fully dismounts and re-renders, re-adding itself to the end of the list but still logically "first" within the DOM content :thinking: Using the stable ID would be a little more reliable if that is the case.

Comment (sarayourfriend) [inline src/components/VMediaInfo/VCopyLicense.vue:17]: Given tabs have an ID already (the value passed to `tab` prop) why do we use the index? It seems potentially flaky if a Tab component fully dismounts and re-renders, re-adding itself to the end of the list but still logically "first" within the DOM content :thinking: Using the stable ID would be a little more reliable if that is the case.

Comment (obulat) [reply src/components/VMediaInfo/VCopyLicense.vue]: You are right. The library I based this implementation off allows for very flexible arrangement of tabs and panels, but it's not really working on SSR.

File: test/unit/specs/components/v-tabs.spec.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ const TestVTabsComponent = (manual = false) => {
     data() {
       return { manual }
     },
-    template: `<VTabs label="tabs" :manual="manual"><template #tabs>
+    template: `<VTabs label="tabs" :manual="manual" selected-id="1"><template #tabs>
```
Comment (sarayourfriend) [inline test/unit/specs/components/v-tabs.spec.js:31]: This change is still dangling/shouldn't be needed anymore, right?

**Review by sarayourfriend — APPROVED** [review]
LGTM! Tested locally and it works fine. Hopefully this resolves the SSR issues :crossed_fingers:

**Review by dhruvkb — APPROVED** [review]
This works locally and also removes a bunch of code so win-win!

**Comment by zackkrida** [comment]
@obulat I'll merge this while you're 😴 so if it _doesn't_ fix the problem, you'll know as early as possible 😆

**Pull request merged**